lib/controller.rb

```diff
@@ -2,4 +2,8 @@ class Controller
   def initialize(game)
     @game = game
   end
+
+  def play
+    while @game.can_continue_playing? do @game.play_move end
```

Comment:
could perhaps be re-phrased:

```rb
until @game.is_over { @game.play_move }
```

Reply:
Thanks, Matt. I've rephrased it to:
`@game.play_move until @game.is_over?`